webpack-main-dev.config.js

```diff
@@ -0,0 +1,10 @@
+const merge = require('webpack-merge');
+const common = require('./webpack-main.config.js');
```

Comment:
`common` -> `commonConfig`; similarly for the production one.

Reply:
👍